schedule.md

```diff
@@ -1,45 +1,67 @@
+## Friday 11/13
+### Homework:
 
-Homework:
+- Create a new repository on GitHub for your project.
+  - Create `readme.md` file in the root of your repo with a a 1-2 sentence summary of your project idea.
+  - Create a `planning` directory in the root of your repo, containing...
+    - A `wireframes` directory, inside of which are *at least* 2 photos or drawings of wireframes
+    - A `user_stories.md` file, containing at least 7 user stories, *or* a link to Trello with at least 7 user stories.
+- Create an issue on *this* repository with a link the repository you created above by **9:00am on Monday, November 16<sup>th</sup>**.
 
-- Create a repository on GitHub.
-  - Include wireframes, ERD, and user stories in a Readme.md
-- Submit a Project 2 idea by creating an issue on this repository with a link to the above by **9am Monday, August 3rd**.
 
-## Monday 8/3
+## Monday 11/16
+**Attendance will be taken at one-on-ones.**
 
-No attendance taken.
+| Time | Event |
+|------|-------|
+| 1:30-2:30pm  | [Instructor AMA (Nick, Adam & Adrian)](./amas.md)|
 
-## Tuesday 8/4
 
-**Attendance will be taken at 9am**.
+## Tuesday 11/17
 
-You will meet and scrum with your squad and meet with your instructor.
+**Attendance will be taken at Outcomes (12:30pm).**
 
-#### Mini-lesson, 1:30-2:45pm: `has_many_through` associations
+| Time | Event |
+|------|-------|
+| 9:00am | Squad Scrum |
+| 12:30-2:30pm | [Outcomes Studio Tours](https://docs.google.com/spreadsheets/u/1/d/15-qrBUoz72aJxQnNnlZfmG4Akhy25p-AYwtCXQKpGSY/edit#gid=0) |
+| 3:00-4:00pm | Mini-lesson |
 
-## Wednesday 8/5
 
-No attendance taken.
+## Wednesday 11/18
 
-#### Mini-lesson, 2-3pm: User auth using the Devise gem
+**Attendance will be taken at any scheduled one-on-ones.**
 
-## Thursday 8/6
+| Time | Event |
+|------|-------|
+| 1:30-2:30pm  | [Instructor AMA (John, Jesse & Andy)](./amas.md)|
+| 3:00-4:00pm | Mini-lesson |
 
-No attendance taken.
 
-#### Mini-lesson, 2-3pm: ActiveRecord Validations
+## Thursday 11/19
 
-#### [Presentation Submission](./presentations.md) is due @ 11pm
+**Attendance will be taken at Peer Code Review (11:00am) AND any scheduled one-on-ones.**
 
-## Friday 8/7
+| Time | Event |
+|------|-------|
+| 11:00am-12:30pm | Peer Code Review |
+| 1:30-2:30pm  | [Instructor AMA (Erica, Matt & Robin)](./amas.md)|
+| 3:00-4:00pm | Mini-lesson |
 
-**Attendance will be taken at 9am**.
 
-- [Project presentations](presentations.md) begin at 9:30am
-- 1:30 - 4:30
-  - Retro!
+
+## Friday 11/20
+
+**Attendance will be taken at Presentations (1:00pm).**
+
+| Time | Event |
+|------|-------|
+| 11:00am | **[Presentation Video](./presentations.md) due** |
+| 12:30-1:00pm | Catered Lunch |
+| 1:00-4:00pm  | [Project Presentations](presentations.md)|
+| 4:00-4:30pm | Feedback Survey |
+| 4:30-5:00pm | Squad Retro! |
```

Comment:
needs to contain a blurb about overarching one-on-one schedule once we establish it.